README.md

```diff
@@ -95,16 +131,40 @@ The v2 SDK will use GitHub [Issues] to track feature requests and issues with th
 
 **Contributing**. You can open pull requests for fixes or additions to the AWS SDK for Go 2.0 Developer Preview release. All pull requests must be submitted under the Apache 2.0 license and will be reviewed by an SDK team member before being merged in. Accompanying unit tests, where possible, are appreciated.
 
-## License
+## Resources
+
+[Developer guide](https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/welcome.html/) - This document
+is a general introduction on how to configure and make requests with the SDK.
+If this is your first time using the SDK, this documentation and the API
+documentation will help you get started. This document focuses on the syntax
+and behavior of the SDK. The [Service Developer Guide](https://aws.amazon.com/documentation/) 
+will help you get started using specific AWS services.
+
+[SDK API Reference Documentation](https://docs.aws.amazon.com/sdk-for-go/v2/api/) - Use this
+document to look up all API operation input and output parameters for AWS
+services supported by the SDK. The API reference also includes documentation of
+the SDK, and examples how to using the SDK, service client API operations, and
+API operation require parameters.
+
+[Service Documentation](https://aws.amazon.com/documentation/) - Use this
+documentation to learn how to interface with AWS services. These guides are
+great for getting started with a service, or when looking for more 
+information about a service. While this document is not required for coding, 
+services may supply helpful samples to look out for.
+
+[SDK Examples](https://github.com/aws/aws-sdk-go-v2/tree/master/example) -
+Included in the SDK's repo are several hand crafted examples using the SDK
+features and AWS services.
+
+[Forum](https://forums.aws.amazon.com/forum.jspa?forumID=293) - Ask questions, get help, and give feedback
 
-This SDK is distributed under the
-[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0),
-see LICENSE.txt and NOTICE.txt for more information.
+[Issues](https://github.com/aws/aws-sdk-go-v2/issues) - Report issues, submit pull requests, and get involved
```

Comment:
probably should use `[Issues][issues]` for these references that are already defined.

Reply:
Fixed!